firmware/controllers/trigger/trigger_central.cpp

```diff
@@ -38,15 +38,13 @@ WaveChart waveChart;
 static scheduling_s debugToggleScheduling;
 #define DEBUG_PIN_DELAY US2NT(60)
 
-trigger_central_s::trigger_central_s() : hwEventCounters() {
-}
-
-TriggerCentral::TriggerCentral() : trigger_central_s(),
+TriggerCentral::TriggerCentral() :
```

Comment:
hold on why is this here? looks like a separate change?

Reply:
This was an unknown bug in the config generator.  It can't handle `define_constructor` properly. I can break that change out, though.

Reply:
#3640 ready pending green check marks